app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -1010,7 +1010,8 @@ DeviceDelegateOculusVR::StartFrame(const FramePrediction aPrediction) {
   ovrMatrix4f matrix = vrapi_GetTransformFromPose(&m.predictedTracking.HeadPose.Pose);
   vrb::Matrix head = vrb::Matrix::FromRowMajor(matrix.M[0]);
 
-  if (m.renderMode == device::RenderMode::StandAlone) {
+  if (m.renderMode == device::RenderMode::StandAlone &&
+      (m.predictedTracking.Status & VRAPI_TRACKING_STATUS_POSITION_TRACKED)) {
```

Comment:
We have exactly the same code in the OpenXR backend. Mind changing it as well?

Reply:
The problem is that this tracking information (with type `ovrTracking2`) is part of the Oculus API, so I don't know if it can be accessed from `DeviceDelegateOpenXR::State`.

Reply:
In OpenXR we could use `location.locationFlags & XR_SPACE_LOCATION_POSITION_VALID_BIT`

Reply:
Thanks, done.